spec/features/idv/steps/gpo_otp_verification_step_spec.rb

```diff
@@ -67,11 +67,36 @@
     end
 
     context 'No ThreatMetrix result on proofing component' do
+      let(:threatmetrix_review_status) { 'pass' }
```

Comment:
Should this be here?

Reply:
Nope. Sometimes my fingers are too fast in vim :o

Reply:
@matthinz we have a test on line 76 which tests a user logging in and entering the GPO code.  They wouldn't have a running session from requesting GPO so they would encounter the same redirect if it were still a problem.